crates/net/discv4/src/config.rs

```diff
@@ -247,6 +253,18 @@ impl Discv4ConfigBuilder {
         self
     }
 
+    /// Adds multiple trusted nodes
+    pub fn add_trusted_nodes(&mut self, nodes: impl IntoIterator<Item = NodeRecord>) -> &mut Self {
+        self.config.trusted_nodes.extend(nodes);
+        self
+    }
```

Comment:
nice,

would like to see another helper function for `add_trusted_node(NodeRecord)`